pilot/const/common.py

```diff
@@ -29,6 +29,7 @@
     '.next',
     '.DS_Store',
     '__pycache__',
+    "site-packages",
```

Comment:
This fixes problems when people create a Python virtual environment within a project, irrespective of how they name the directory. There's at least a couple of reported case of this exact reason for TokenLimitError, including in telemetry and discord.